src/pages/StartPage.jsx

```diff
@@ -91,7 +91,7 @@ function StartPage({ onLogout }) {
             <p className="current-user-name">{user.secondName}</p>
             <p className="current-user-role">{getRoleTitle(user.role)}</p>
           </div>
-          <LogoutOutlined className="logout-icon" onClick={(e) => {onLogout("user-id");}} />
+          <LogoutOutlined className="logout-icon" onClick={(e) => {onLogout("access-token");}} />
```

Comment:
какое-то странное апи получается. мы передаём в функцию onLogout какую-то строку. давай просто по клику вызывать onLogout

а имя куки в самой функции зашьём
https://github.com/perfomance-review/pr-frontend/blob/fe9c775c88fcd830cb13b914615f54371b279240/src/App.js#L18-L21

и там тоже можно просто  setIsLogin(false);

```suggestion
          <LogoutOutlined className="logout-icon" onClick={onLogout} />
```

Reply:
https://github.com/perfomance-review/pr-frontend/blob/fe9c775c88fcd830cb13b914615f54371b279240/src/API/Cookies.js#L8-L10

у функции deleteCookie второй параметр со значением бесполезен. мы всё равно куку удаляем

сейчас там вообще userId почему-то передаётся https://github.com/perfomance-review/pr-frontend/blob/fe9c775c88fcd830cb13b914615f54371b279240/src/App.js#L19